### DISCUSSION TITLE
Adopting the Bot for different communites - Licensing

### DISCUSSION THREAD

**Discussion by rapus95** [Humans-of-Julia/HoJBot.jl#46] [General]
I'm thinking about adopting the HoJ Bot for a different (single-language) community, since some of the commands here won't make sense there and most from there won't make sense here. So I wonder, what are the things aside of the MIT License that I need to carry with it in order to not break any licensing etc?

**Comment by tk3369** [comment]
The MIT license is extremely simple. You just have to include a copy of the license and you can do whatever you want with it. 

My other thought is that we can make the bot more customizable to enable/disable features. Would that work for you? The only downside of this approach is that you cannot add features that are supposed to be private to your site.

**Reply by rapus95** [reply]
given that the target server is entirely in german almost all features aren't needed there and we'd need localization support 😂 🙈 So I'm not sure if it's really worth the effort 🙈

**Reply by Azzaare** [reply]
One thing that could be done is to create a branch for that bot, and you can rebase on the main branch when important changes occur.
We can make the features activated through keyword arg when we launch the bot.

How does it sound?
